lib/block-supports/typography.php

```diff
@@ -403,8 +403,14 @@ function gutenberg_get_computed_fluid_typography_value( $args = array() ) {
 
 	// Build CSS rule.
 	// Borrowed from https://websemantics.uk/tools/responsive-font-calculator/.
-	$view_port_width_offset = round( $minimum_viewport_width['value'] / 100, 3 ) . $font_size_unit;
-	$linear_factor          = 100 * ( ( $maximum_font_size['value'] - $minimum_font_size['value'] ) / ( $maximum_viewport_width['value'] - $minimum_viewport_width['value'] ) );
+	$view_port_width_offset    = round( $minimum_viewport_width['value'] / 100, 3 ) . $font_size_unit;
+	$linear_factor_denominator = $maximum_viewport_width['value'] - $minimum_viewport_width['value'];
+
+	/*
+	 * If the linear factor denominator is 0, we cannot calculate a fluid value.
+	 * Setting $linear_factor to 0 will cause $linear_factor_scaled to default to 1.
+	 */
+	$linear_factor          = ! empty( $linear_factor_denominator ) ? 100 * ( ( $maximum_font_size['value'] - $minimum_font_size['value'] ) / ( $linear_factor_denominator ) ) : 0;
 	$linear_factor_scaled   = round( $linear_factor * $scale_factor, 3 );
 	$linear_factor_scaled   = empty( $linear_factor_scaled ) ? 1 : $linear_factor_scaled;
```

Comment:
Would this be more readable?

```php

if ( ! empty( $linear_factor_denominator ) ) {
    $linear_factor = 100 * ( ( $maximum_font_size['value'] - $minimum_font_size['value'] ) / ( $linear_factor_denominator ) );
    $linear_factor_scaled = round( $linear_factor * $scale_factor, 3 );
} else {
    $linear_factor_scaled = 1;
}

$linear_factor_scaled = empty( $linear_factor_scaled ) ? 1 : $linear_factor_scaled;
```

Reply:
Slightly! That looks a bit more readable to me since it reduces a bit of the complexity in a single line 👍

I don't feel strongly about it either way, though!